src/Symfony/Component/Validator/Test/ConstraintValidatorTestCase.php

```diff
@@ -60,6 +60,7 @@ abstract class ConstraintValidatorTestCase extends TestCase
     protected $propertyPath;
     protected $constraint;
     protected $defaultTimezone;
+    protected string $locale;
```

Comment:
the property should be defined as `protected ?string $locale = null`, or the check line 124 needs to be changed

Reply:
I agree, thanks.